community-meetings/2021-08-10.md

```diff
@@ -0,0 +1,27 @@
+# Flatcar community call Tuesday, 10th of August, 5:30 pm CEST
+
+- Zoom link for active participation: [https://us06web.zoom.us/j/83689202160](https://us06web.zoom.us/j/83689202160)
+- Youtube stream for passively watching: [Link will become available shortly before the meeting]
+
+## Welcome
+- Brief intro / check-in of all participants in the Zoom call. Please introduce yourself and share what brings you here today.
+- Review the meeting agenda.
+
+## News
+- New section! We briefly discuss news and happenings in the Flatcar world.
+
+## Spotlight: Docker 20.10 / cgroups v2 update
+- Docker 20 and CGroups v2 (unified mode) are coming to Flatcar Linux! We'll give an overview of the challenges and the implications, as well as discuss the timeline.
+
+## Spotlight community committer: fixing the Locksmith reboot issue
```

Comment:
As mentioned last week, I think the spotlight will be more around :arrow_down: 
```suggestion
## Spotlight community committer: contributing to Flatcar
```

and not dedicated to the locksmith issue. :)